lib/geordi/cucumber.rb

```diff
@@ -4,69 +4,33 @@
 # This require-style is to prevent Ruby from loading files of a different
 # version of Geordi.
 require File.expand_path('interaction', __dir__)
-require File.expand_path('firefox_for_selenium', __dir__)
 require File.expand_path('settings', __dir__)
 
 module Geordi
   class Cucumber
 
-    VNC_DISPLAY = ':17'.freeze
-    VNC_PASSWORD_FILE = File.expand_path('~/.vnc/passwd').freeze # default for "vncpasswd"
-    VNC_SERVER_DEFAULT_OPTIONS = "-localhost -nolisten tcp -geometry 1280x1024 -rfbauth #{VNC_PASSWORD_FILE}".freeze
-    VNC_SERVER_COMMAND = "vncserver #{VNC_DISPLAY} #{ENV.fetch('GEORDI_VNC_OPTIONS', VNC_SERVER_DEFAULT_OPTIONS)}".freeze
-    VNC_VIEWER_COMMAND = "vncviewer -passwd #{VNC_PASSWORD_FILE}".freeze
-    VNC_ENV_VARIABLES = %w[DISPLAY BROWSER LAUNCHY_BROWSER].freeze
```

Comment:
Where did all the logic for running tests inside VNC go? Was it ported to our legacy project docker setup?

Reply:
The legacy docker is running with a old geordi version, so running tests inside VNC is handled by two instances really:
- The docker container is starting a VNC server with a passwd file located in the containers root directory.
- This is triggered by running tests from the containers shell (e.g via `geordi cucumber feature/feature_xy.feature` 
- The VNC viewer can be started via `geordi docker vnc` from the local shell, opening a VNC window connected to the VNC server in the docker container.